src/constants/externalResources.ts

```diff
@@ -38,9 +38,6 @@ export const getJiraImprovementUrl = (jiraHost: string) =>
 export const konamiSoundTrackUrl =
   "https://www.myinstants.com/media/sounds/mvssf-win.mp3";
 
-export const githubMergeQueueUrl =
```

Comment:
Should we include the documentation before we launch to reduce any potential confusion? @bsamek

Reply:
Pushed to https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Merge-Queue (will appear in a few minutes).